src/cowrie/commands/wget.py

```diff
@@ -157,8 +157,7 @@ def download(self, url, fakeoutfile, outputfile, *args, **kwargs):
             out_addr = (CowrieConfig().get('honeypot', 'out_addr'), 0)
 
         if scheme == b'https':
-            contextFactory = ssl.ClientContextFactory()
-            contextFactory.method = SSL.SSLv23_METHOD
+            contextFactory = ssl.CertificationOptions(method=SSL.SSLv23_METHOD)
```

Comment:
Hi! I recently re-built cowrie again, and when running and trying to wget some file from an HTTPS site, I get an error saying that `twisted.internet.ssl` has no attribute `CertificationOptions`... Maybe you meant [CertificateOptions](https://twistedmatrix.com/documents/current/api/twisted.internet.ssl.CertificateOptions.html)? This seems to be a bug introduced with this PR, hence why I'm writing here...